docs/topics/contributing-repository-filesystem-layout.adoc

```diff
@@ -16,7 +16,7 @@ NOTE: The triple-dot indicates there are more files or directories in the partic
 │   │   │   ├── document-attributes.adoc <6>
 │   │   │   ├── ...
 │   │   │   ├── wildfly-swarm <7>
-│   │   ├── configmap-mission-design-tradeoffs.adoc
+│   │   ├── mission-design-tradeoffs.adoc
```

Comment:
I'd rather we included a name of a file that actually exists here (doesn't have to be ConfigMap-related).